.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,28 @@
+# .github/workflows/chromatic.yml
+
+# Workflow name
+name: 'Chromatic'
+
+# Event for the workflow
+on: push
+
+# List of jobs
+jobs:
```

Comment:
Do you want to run it on each push? Maybe only on PR's to develop.

Also we can utilize github pages to deploy the storybook instead of Chromatic

Reply:
If in this PR we can go with chromatic